settings/settings-url.php

```diff
@@ -188,6 +188,7 @@ protected function redirect_lang() {
 			<?php
 			// That's nice to display the right home urls but don't forget that the page on front may have no language yet
 			$lang = $this->model->post->get_language( $this->page_on_front );
+			/** @var PLL_Language $lang */
 			$lang = $lang ? $lang : $this->model->get_language( $this->options['default_lang'] );
```

Comment:
```suggestion
			$lang = $lang instanceof PLL_Language ? $lang : $this->model->get_language( $this->options['default_lang'] );
```
To remove previous inline type.

Reply:
Oh I see... THX